devicon.json

```
@@ -5194,6 +5194,27 @@
         "color": "#000000",
         "aliases": []
     },
+    {
+        "name": "okta",
+        "altnames": [
+            "okta developer"
+        ],
+        "tags": [
+            "auth"
```

What do you think about this?
```suggestion
            "auth",
            "security"
```

---

sure, makes sense